api/src/quest/application/combined-course-route.js

```diff
@@ -117,6 +118,17 @@ const register = async function (server) {
               number: Joi.number().integer().empty('').default(1),
               size: Joi.number().integer().empty('').default(10),
             },
+            filters: {
+              fullName: Joi.string().empty('').optional(),
```

Comment:
on pourrait mettre une limite de taille


```suggestion
  fullName: Joi.string().empty('').max(255).optional(),
```